src/util/Timer.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @flow
+ * @file Timer Class
+ * @author Box
+ */
+
+class Timer {
```

Comment:
Can we get rid of this file and just use performance?

Reply:
I don't see why not :)